variables.tf

```
@@ -77,6 +77,25 @@ variable "minio_service" {
   description = "Minio data-object contains service_name, port, access_key and secret_key"
 }
 
+variable "minio_vault_secret" {
+  type = object({
+    use_vault_provider     = bool,
+    vault_kv_policy_name   = string,
+    vault_kv_path          = string,
+    vault_kv_access_key    = string,
+    vault_kv_secret_key    = string
```

The current names make it sound like it is actually the values, but we are talking about the names here, correct?

```suggestion
    vault_kv_access_key_name    = string,
    vault_kv_secret_key_name    = string
```

---

It is kind of tricky, due we do not have consistency among the modules, examples
- [terraform-nomad-minio variables](https://github.com/fredrikhgrelland/terraform-nomad-minio/blob/master/variables.tf#L103-L104)
- [terraform-nomad-postgres variables](https://github.com/fredrikhgrelland/terraform-nomad-postgres/blob/master/variables.tf#L85-L86)

---

Right.. Fair point. Could that be something we gradually change though? I don't like the current names, gives a strong impression that they are the actual values

---

> Right.. Fair point. Could that be something we gradually change though? I don't like the current names, gives a strong impression that they are the actual values

I agree. Suggest you open an issue and give your thoughts and suggestion for variable names to be used for every module.

---

Issue registered https://github.com/fredrikhgrelland/terraform-nomad-minio/issues/87